p2p/node/helpers.go

```diff
@@ -34,14 +34,12 @@ func GenerateTestNodeWithConfig(t *testing.T) (LocalNode, *Info) {
 
 	addr := net.TCPAddr{IP: net.ParseIP("0.0.0.0"), Port: tcpPort}
 
-	var localNode LocalNode
-
-	localNode, err = NewNodeIdentity()
+	ln, err = NewNodeIdentity()
```

Comment:
Re-use named return which stops the code from needing an "emptyNode" variable in the package.